### PR TITLE
Add cell types

### PR DESCRIPTION
**Pull request by IvanRodriCalleja**
- Add BooleanCell
- Add DateCell
- Add NumberCell
- Add StringCell

### CODE REVIEW

File: src/grid/cell/cellTypes.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const CELL_TYPES = {
```
Comment (acnagrellos) [inline src/grid/cell/cellTypes.js]: I prefer create a grid configuration file and put this cell types in it.

Comment (IvanRodriCalleja) [reply src/grid/cell/cellTypes.js]: Do you mean to set all possible enums or constants in the same file?

Comment (acnagrellos) [reply src/grid/cell/cellTypes.js]: Nope. I think only the CELL_TYPES definition should be in this configuration file.

Comment (IvanRodriCalleja) [reply src/grid/cell/cellTypes.js]: This change i have in other branch. I am going to create a PR with this changes to this branch and next merge.

File: .eslintrc [hunk 2]
```diff
@@ -20,6 +20,7 @@
   "rules": {
     "space-before-function-paren": 0,
     "react/jsx-boolean-value": 0,
+    "react/prop-types": 0,
```
[the remark this answers is not in the document]
Comment (IvanRodriCalleja) [reply .eslintrc]: Yes

File: src/grid/Cell.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React, { memo } from 'react'
+
+export const Cell = memo(({ columnIndex, rowIndex, style }) => (
```
[the remark this answers is not in the document]
Comment (IvanRodriCalleja) [reply src/grid/Cell.js]: It is removed in other branch

**Pull request closed without merging**